.gitignore

```
@@ -0,0 +1,7 @@
+# Claude Code files
+CLAUDE.md
+helperfilebom.md
+ExoGuitarScripts.md
+
+# Generated files
+MISSING_DOCS_CHECKLIST.md
```

exclude generated files from Git. 
```suggestion
MISSING_DOCS_CHECKLIST.md
docs/public/data
docs/public/images

```